[docs] Document how to manage EAS Workflow runs (#48928)
expo/expo: docs/pages/eas/workflows/manage-workflow-runs.mdx

---
title: Manage EAS Workflow runs
sidebar_title: Manage workflow runs

> description: View, cancel, and re-run EAS Workflow runs.

Reviewer: ✏️ **style** `terminology.md#expo-product-and-feature-names`

The product name is always plural, so "EAS Workflow runs" should be "EAS Workflows". Nothing under **pages/eas** uses the singular form today, and the house phrasing for the thing itself is a lowercase "workflow run".

```suggestion
title: Manage workflow runs in EAS Workflows
sidebar_title: Manage workflow runs
description: Learn how to view, cancel, and re-run workflow runs in EAS Workflows.
```

<!-- docs-review -->

---

> Each time a workflow is triggered, EAS creates a workflow run. You can use the EAS dashboard to follow the run, inspect its jobs, cancel work that is in progress, or create a new run from a completed attempt.

Reviewer: ✏️ **style** `prose.md#core-principles`

The second sentence carries four clauses in 28 words, and "attempt" is a second name for what the rest of the page calls a run. Split the sentence and use "run" throughout, including "the dashboard links the attempts" on line 32.

```suggestion
Each time a workflow is triggered, EAS creates a workflow run. Use the EAS dashboard to follow a run and inspect its jobs. You can also cancel a run that is in progress, or create a new run from one that already finished.
```

<!-- docs-review -->

## Workflow run statuses

A workflow run has one of the following statuses:

- **New**: EAS created the run, but it has not started yet.
- **In progress**: One or more jobs are running.
- **Action required**: A job is waiting for a manual action, such as an approval.
- **Success**: All jobs completed successfully. This status is terminal.
- **Failure**: At least one job failed. This status is terminal.

> - **Canceled**: The run was canceled before it completed. This status is terminal.

Reviewer: 💡 **nit** `review-ai-text.md#t5-restatement`

"This status is terminal" appears three times; say it once for the three statuses that share it.

<!-- docs-review -->

> For the status values returned by the EAS REST API, see [Get a workflow run](/eas/workflows/rest-api/#get-a-workflow-run).

Reviewer: Either way is fine. Though, we don't use quotes and had our docs cleanup over the years to avoid using quotes in inlined links. Whatever we decide, we should follow the pattern for each link. Right now, I think not using them, also aligns with TSDoc in SDK docs that have inlined links.
